gslib/tests/test_mb.py

```diff
@@ -105,3 +105,27 @@ def test_create_with_retention_on_s3_urls_fails(self):
         return_stderr=True)
     self.assertRegexpMatches(
         stderr, r'Retention policy can only be specified for GCS buckets.')
+
+  def test_create_with_pap_enforced(self):
```

Comment:
Tests are not skipped for XML and hence they are failing

https://fusion2.corp.google.com/invocations/f6cb842e-142b-4bbf-a8ea-394be7d5c17b/targets/cloud_storage_gsutil%2Flinux%2Fpy37_xml/log